src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/trial_of_the_champion.cpp

```diff
@@ -15,56 +15,262 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-/* ScriptData
-SDName: Trial Of the Champion
-SD%Complete:
-SDComment:
-SDCategory: trial_of_the_champion
-EndScriptData */
-
-/* ContentData
-npc_announcer_toc5
-EndContentData */
-
-#include "ScriptMgr.h"
 #include "GameObject.h"
-#include "InstanceScript.h"
-#include "Map.h"
-#include "MotionMaster.h"
-#include "ObjectAccessor.h"
-#include "Player.h"
+#include "ScriptMgr.h"
 #include "ScriptedCreature.h"
 #include "ScriptedGossip.h"
-#include "TemporarySummon.h"
 #include "trial_of_the_champion.h"
 #include "Vehicle.h"
+#include "Player.h"
+
+#define GOSSIP_START_EVENT1     "I am ready."
+#define GOSSIP_START_EVENT2     "I am ready for the next challenge."
+#define GOSSIP_START_EVENT_SKIP "I am ready. However I'd like to skip the pageantry."
```

Comment:
It would be very nice if you can convert these `#define` lines to DB text and use DB references instead of hardcoded text.

Reply:
I dont know yet, but ill research, it doesnt sound that hard

Reply:
Whats the right way to commit DB changes? an SQL file on the commit?

Reply:
Yes, add SQL files to the same branch as this PR so the content can be checked in travis-ci too.

Reply:
use something like `9999_99_99_world_TOC5.sql` so it's easy to find in the folder